### PR TITLE
UI/send page

### PR DESCRIPTION
**Pull request by shiauweitsung**
## send flow ui update

#9 
1.send view ui update
2.add send to from & to input component
3.send amount ui update
4.base style add class name
5.button add hover effect

### CODE REVIEW

File: app/manifest/_base.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "512": "images/icon-512.png"
   },
   "manifest_version": 2,
-  "name": "__MSG_appName__",
+  "name": "Amino",
```
Comment (mayone) [inline app/manifest/_base.json]:
```suggestion
  "name": "__MSG_appName__",
```

File: ui/components/app/app-header/app-header.component.js [hunk 2]
```diff
@@ -110,12 +110,16 @@ export default class AppHeader extends PureComponent {
       disabled,
       onClick,
     } = this.props;
+    const pathname = history?.location?.pathname;
```
Comment (mayone) [inline ui/components/app/app-header/app-header.component.js]:
```suggestion
```

File: ui/components/app/app-header/app-header.component.js [hunk 3]
```diff
@@ -110,12 +110,16 @@ export default class AppHeader extends PureComponent {
...
+    const logoShow = pathname === UNLOCK_ROUTE;
```
Comment (mayone) [inline ui/components/app/app-header/app-header.component.js]:
```suggestion
```

File: ui/pages/send/send-content/send-from/send-from.component.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+function SendFrom({ accounts, selectedAddress, accountsList }) {
+  const balanceValue = accounts[selectedAddress]
+    ? accounts[selectedAddress].balance
+    : '';
+  const t = useI18nContext();
+  return (
+    <div className={classnames('send-v2__from-wrap')}>
+      <div className="send-v2__from-title">from</div>
+      <div className="send-v2__from-cont">
+        <div className="send-v2__from-cont-img">pic</div>
```
Comment (mayone) [inline ui/pages/send/send-content/send-from/send-from.component.js]: Should be `Identicon` for now

File: ui/pages/send/send-content/send-to/send-to.component.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+function SendTo({}) {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const recipient = useSelector(getRecipient);
+  return (
+    <div className={classnames('send-v2__to-wrap')}>
+      <div className="send-v2__to-title">To</div>
+      <div className="send-v2__to-cont">
+        <div className="send-v2__to-cont-img">pic</div>
```
Comment (mayone) [inline ui/pages/send/send-content/send-to/send-to.component.js]: - Should be `Identicon` for now

File: ui/pages/send/send-content/send-from/send-from.component.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+function SendFrom({ accounts, selectedAddress, accountsList }) {
+  const balanceValue = accounts[selectedAddress]
+    ? accounts[selectedAddress].balance
+    : '';
+  const t = useI18nContext();
+  return (
+    <div className={classnames('send-v2__from-wrap')}>
+      <div className="send-v2__from-title">from</div>
```
Comment (mayone) [inline ui/pages/send/send-content/send-from/send-from.component.js]:
```suggestion
      <div className="send-v2__from-title">From</div>
```

File: ui/components/ui/icon/help-outline-icon.component.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function HelpOutLineIcon({
```
Comment (mayone) [inline ui/components/ui/icon/help-outline-icon.component.js]:
```suggestion
export default function HelpOutlineIcon({
```

File: ui/components/ui/icon/help-outline-icon.component.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        fill={color}
+      />
+    </svg>
+  );
+}
+
+HelpOutLineIcon.propTypes = {
```
Comment (mayone) [inline ui/components/ui/icon/help-outline-icon.component.js]:
```suggestion
HelpOutlineIcon.propTypes = {
```

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 9]
```diff
@@ -60,6 +60,7 @@ import {
 
 import Typography from '../../components/ui/typography/typography';
 import { MIN_GAS_LIMIT_DEC } from '../send/send.constants';
+import HelpOutLineIcon from '../../components/ui/icon/help-outline-icon.component';
```
Comment (mayone) [inline ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]:
```suggestion
import HelpOutlineIcon from '../../components/ui/icon/help-outline-icon.component';
```

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 10]
```diff
@@ -468,20 +471,21 @@ export default class ConfirmTransactionBase extends Component {
...
                         </a>
-                      </p>
+                      </p> */}
                     </>
                   }
                   position="top"
                 >
-                  <i className="fa fa-info-circle" />
+                  {/* <i className="fa fa-info-circle" /> */}
+                  <HelpOutLineIcon />
```
Comment (mayone) [inline ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]:
```suggestion
                  <HelpOutlineIcon />
```

File: ui/components/ui/info-tooltip/info-tooltip.js [hunk 11]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Tooltip from '../tooltip';
+import HelpOutLine from '../icon/help-outline-icon.component';
```
Comment (mayone) [inline ui/components/ui/info-tooltip/info-tooltip.js]:
```suggestion
import HelpOutlineIcon from '../icon/help-outline-icon.component';
```

File: ui/components/ui/info-tooltip/info-tooltip.js [hunk 12]
```diff
@@ -34,7 +35,8 @@ export default function InfoTooltip({
         html={contentText}
         theme={wide ? 'tippy-tooltip-wideInfo' : 'tippy-tooltip-info'}
       >
-        <InfoTooltipIcon fillColor={iconFillColor} />
+        {/* <InfoTooltipIcon fillColor={iconFillColor} /> */}
+        <HelpOutLine color="#222222" />
```
Comment (mayone) [inline ui/components/ui/info-tooltip/info-tooltip.js]:
```suggestion
        <HelpOutlineIcon color="#222222" />
```

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js [hunk 13]
```diff
@@ -5,12 +5,21 @@ import {
...
+import { getMetaMaskAccounts } from '../../../../selectors';
 import SendAmountRow from './send-amount-row.component';
 
 export default connect(mapStateToProps, mapDispatchToProps)(SendAmountRow);
 
 function mapStateToProps(state) {
+  const {
+    metamask: { nativeCurrency },
+  } = state;
+  const { metamask } = state;
```
Comment (mayone) [inline ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]:
```suggestion
```

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js [hunk 14]
```diff
@@ -5,12 +5,21 @@ import {
...
   return {
+    metamask,
```
Comment (mayone) [inline ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]:
```suggestion
```
- seems not needed?

File: app/manifest/_base.json (hunk 1 above)
Comment (shiauweitsung) [reply app/manifest/_base.json]: fixed in 66ce7f463bb930ba2978b71767c297777aaa69b9

File: ui/components/app/app-header/app-header.component.js (hunk 2 above)
Comment (shiauweitsung) [reply ui/components/app/app-header/app-header.component.js]: fixed in 66ce7f463bb930ba2978b71767c297777aaa69b9

File: ui/components/app/app-header/app-header.component.js (hunk 3 above)
Comment (shiauweitsung) [reply ui/components/app/app-header/app-header.component.js]: fixed in 66ce7f463bb930ba2978b71767c297777aaa69b9

File: ui/pages/send/send-content/send-from/send-from.component.js (hunk 4 above)
Comment (shiauweitsung) [reply ui/pages/send/send-content/send-from/send-from.component.js]: fixed in bd5a9d52dfae71fc07e4aa91d9c7d3d7ede5c444

File: ui/pages/send/send-content/send-to/send-to.component.js (hunk 5 above)
Comment (shiauweitsung) [reply ui/pages/send/send-content/send-to/send-to.component.js]: fixed in bd5a9d52dfae71fc07e4aa91d9c7d3d7ede5c444

File: ui/pages/send/send-content/send-from/send-from.component.js (hunk 6 above)
Comment (shiauweitsung) [reply ui/pages/send/send-content/send-from/send-from.component.js]: fixed in bd5a9d52dfae71fc07e4aa91d9c7d3d7ede5c444

File: ui/components/ui/icon/help-outline-icon.component.js (hunk 7 above)
Comment (shiauweitsung) [reply ui/components/ui/icon/help-outline-icon.component.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/components/ui/icon/help-outline-icon.component.js (hunk 8 above)
Comment (shiauweitsung) [reply ui/components/ui/icon/help-outline-icon.component.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js (hunk 9 above)
Comment (shiauweitsung) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js (hunk 10 above)
Comment (shiauweitsung) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/components/ui/info-tooltip/info-tooltip.js (hunk 11 above)
Comment (shiauweitsung) [reply ui/components/ui/info-tooltip/info-tooltip.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/components/ui/info-tooltip/info-tooltip.js (hunk 12 above)
Comment (shiauweitsung) [reply ui/components/ui/info-tooltip/info-tooltip.js]: fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js (hunk 14 above)
Comment (shiauweitsung) [reply ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]: fixed in 9f54bd84a91fb9d310746a53230f2ebdc9ecfe50

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js (hunk 13 above)
Comment (shiauweitsung) [reply ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]: fixed in 9f54bd84a91fb9d310746a53230f2ebdc9ecfe50

File: app/manifest/_base.json (hunk 1 above)
Comment (mayone) [reply app/manifest/_base.json]: - Verified

File: ui/components/app/app-header/app-header.component.js (hunk 2 above)
Comment (mayone) [reply ui/components/app/app-header/app-header.component.js]: - Verified

File: ui/components/app/app-header/app-header.component.js (hunk 3 above)
Comment (mayone) [reply ui/components/app/app-header/app-header.component.js]: - Verified

File: ui/pages/send/send-content/send-from/send-from.component.js (hunk 4 above)
Comment (mayone) [reply ui/pages/send/send-content/send-from/send-from.component.js]: - Verified

File: ui/pages/send/send-content/send-to/send-to.component.js (hunk 5 above)
Comment (mayone) [reply ui/pages/send/send-content/send-to/send-to.component.js]: - Verified

File: ui/pages/send/send-content/send-from/send-from.component.js (hunk 6 above)
Comment (mayone) [reply ui/pages/send/send-content/send-from/send-from.component.js]: - Verified

File: ui/components/ui/icon/help-outline-icon.component.js (hunk 8 above)
Comment (mayone) [reply ui/components/ui/icon/help-outline-icon.component.js]: - Verified

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js (hunk 9 above)
Comment (mayone) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]: - Verified

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js (hunk 10 above)
Comment (mayone) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js]: - Verified

File: ui/components/ui/info-tooltip/info-tooltip.js (hunk 11 above)
Comment (mayone) [reply ui/components/ui/info-tooltip/info-tooltip.js]: - Verified

File: ui/components/ui/info-tooltip/info-tooltip.js (hunk 12 above)
Comment (mayone) [reply ui/components/ui/info-tooltip/info-tooltip.js]: - Verified

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js (hunk 13 above)
Comment (mayone) [reply ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]: - Verified

File: ui/pages/send/send-content/send-amount-row/send-amount-row.container.js (hunk 14 above)
Comment (mayone) [reply ui/pages/send/send-content/send-amount-row/send-amount-row.container.js]: - Verified

File: ui/components/ui/icon/help-outline-icon.component.js (hunk 7 above)
Comment (mayone) [reply ui/components/ui/icon/help-outline-icon.component.js]: - Verified

**Comment by mayone** [comment]
- lgtm, @shiauweitsung  please squash and merge

**Pull request merged**